src/nifgen/metadata/enums.py

```diff
@@ -33,6 +33,46 @@
             }
         ]
     },
+    'ArbitrarySequenceHandle': {
```

Comment:
Why are we adding all of these new Enums?

Reply:
Most of them are referenced as a `grpc_enum` in some existing function or attribute. So we don't technically need them _yet_, but will soon.

There are a few, though, that were only referenced in 'codegen_method':'no' attributes, so I can mark those enums a 'codegen_method':'no', too.

Reply:
Why does grpc require us to add new enums?

Reply:
Actually, when there's not a corresponding python enum, we probably won't need that to generate the enum for `grpc_enum` functions/attributes. (We just need to know that it's defined as _some_ enum for grpc.) So I could technically mark those enums as 'codegen_method':'no', too, if you insist.

Reply:
Note that even with the enums added to the metadata, they still don't appear in the generated API (as you can see in the [diffs](https://github.com/ni/nimi-python/pull/1819/commits/dba97eb5194d0c741fb0c27b2f5de61812ae713a)), because the nimi-python codegen step throws out any unreferenced enums.

Reply:
I hadn't noticed that. I guess that's okay for now, but I think I still would like to mark them as 'codegen_method':'no' as soon as possible. It's confusing to list unused enums.

Reply:
> but I would lean towards not adding

This PR shouldn't be changing customer-facing API.

Reply:
I have a [PR](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/373719) ready to commit that will allow this. After that, I'll go apply it to all of those enums. But I'll make those changes separate from this PR.